### PR TITLE
Completed html-forms exercise.

### PR DESCRIPTION
**Pull request by mpowers1113**

### CODE REVIEW

**Comment by mpowers1113** [comment]
Screen Shot 2021-10-05 at 3 53 32 PM

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Document</title>
+  </head>
+  <body>
+  <h1>Coding Form!</h1>
+    <form action="www.codingform.com/why.php">
+      <p>
+        <label for="email">Email</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: This `label` isn't configured correctly, the value of the `for` attribute on a `label` should match the value of the `id` attribute on the form control element it is paired with.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <input type="email" name="email"/>
+      </p>
+      <p>
+        <label for="coding-is-cool">Why is coding cool?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:14]: Same here, this is not configured correctly to pair with its form control element

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <input type="radio" value="HTML" name="favorite-lang" id="HTML"/>
+        <label for="HTML">HTML</label>
+        <input type="radio" value="CSS" name="favorite-lang" id="CSS"/>
+        <label for="CSS">CSS</label>
+        <input type="radio" value="JavaScript" name="favorite-lang" id="JavaScript"/>
+        <label for="JavaScript">JavaScript</label>
+      </p>
+      <p>Would you like to sign up for more information?
+        <input type="checkbox" value="subscribe" id="subscribe" name="subscribe"/>
+        <label name="subscribe">Sign me up!</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:32]: Same here, this is not configured correctly to pair with its form control element

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      </p>
+      <p>
+        <label for="comments">Comments</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: Same here, this is not configured correctly to pair with its form control element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here!! A few notes for you:

**Comment by mpowers1113** [comment]
Screen Shot 2021-10-05 at 5 10 38 PM

**Comment by mpowers1113** [comment]
reformatted with div's instead of paragraph elements
Screen Shot 2021-10-05 at 5 16 58 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**